test/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    "env": {
+        "es6": true,
+		"mocha": true,
+		"amd": true
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "off"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+		"brace-style": [
```

Comment:
Tabs -> spaces. Does eslint not crawl up and see the one in the parent directory, though?